x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -523,6 +524,87 @@ export const getEndpointConsoleCommands = ({
       privileges: endpointPrivileges,
     }),
   });
+  if (crowdstrikeRunScriptEnabled) {
+    consoleCommands.push({
+      name: 'runscript',
+      about: getCommandAboutInfo({
+        aboutInfo: CROWDSTRIKE_CONSOLE_COMMANDS.runscript.about,
+        isSupported: doesEndpointSupportCommand('runscript'),
+      }),
+      RenderComponent: () => null,
+      meta: {
+        agentType,
+        endpointId: endpointAgentId,
+        capabilities: endpointCapabilities,
+        privileges: endpointPrivileges,
+      },
+      exampleUsage: `runscript -Raw=\`\`\`Get-ChildItem .\`\`\` -CommandLine=""`,
```

Comment:
1. Changing `-` into `--` thanks!
2. I'd say it could support both, or even more: `runscript --Raw=ls` | `runscript --Raw="ls"` | `runscript --Raw='ls' ` | 'runscript --Raw=`ls`'